### ISSUE TITLE
"The Onset" not completing

### ISSUE THREAD

**Issue by Lesani** [Anuken/Mindustry#8522]
**Platform**: *The type of device you were playing on - Android/iOS/Mac/Windows/Linux* ("All" is NOT a platform!)
Windows/Steam

**Build**: *The build number under the title in the main menu. Required. "LATEST" IS NOT A VERSION, I NEED THE EXACT BUILD NUMBER OF YOUR GAME.*
143.1

**Issue**: *Explain your issue in detail.*
First time player, did the tutorial level "The Onset", killed, disassembed etc everything, put a miner on every ressource, still not completing.
- I have researched everything available to me
- there are no more enemies or structures
- I have tried closing and reopening the game, still I can only play "The Onset"

![image](https://user-images.githubusercontent.com/5182104/233427990-cd07ceb0-c62c-4994-a1aa-de7b8d0caf30.png)

![image](https://user-images.githubusercontent.com/5182104/233427903-ade796ec-69ed-4d1a-8b45-72ba7238f61f.png)

![image](https://user-images.githubusercontent.com/5182104/233428987-fd44b0c2-c7f5-45b3-82e2-1e384fcf82f1.png)

**Steps to reproduce**: *How you happened across the issue, and what exactly you did to make the bug happen.*
just played the tutorial?

**Link(s) to mod(s) used**: *The mod repositories or zip files that are related to the issue, if applicable.*
vanilla

**Save file**: *The (zipped) save file you were playing on w
hen the bug happened. THIS IS REQUIRED FOR ANY ISSUE HAPPENING IN-GAME OR IN MULTIPLAYER, REGARDLESS OF WHETHER YOU THINK IT HAPPENS EVERYWHERE. DO NOT DELETE OR OMIT THIS LINE UNLESS YOU ARE SURE THAT THE ISSUE DOES NOT HAPPEN IN-GAME. IF YOU DO NOT HAVE A SAVE, DON'T WASTE TIME OPENING THIS ISSUE.*

top-tip: Instead of doing the all-caps nightmare above maybe hint how to export the save file so a new player like me doesn't have to search reddit how to get save files [tutorial.zip](https://github.com/Anuken/Mindustry/files/11288363/tutorial.zip)

If you remove the line above without reading it properly and understanding what it means, I will reap your soul. Even if you're playing on someone's server, you can still save the game to a slot.

**(Crash) logs**: *Either crash reports from the crash folder, or the file you get when you go into Settings -> Game Data -> Export Crash logs. REQUIRED if you are reporting a crash.*

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [ ] **I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.** 
- [ x] I have updated to the latest release available on steam, which has the same build number as the latest release on github
- [x ] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by DX-36** [comment]
look around the planet for the next sector

**Comment by Anuken** [comment]
You have another sector you can launch to on the planet.

**Comment by Lesani** [comment]
![image](https://user-images.githubusercontent.com/5182104/233805582-389d00a8-8210-48b7-b91b-03ee6e6da2df.png)
well, it's not like I am new to games, and I even moved the planet a little, but not enough it seems.
after all the coddling and guiding the game does the whole first level, one has to 1: assume the planet can be turned and 2: that the 2nd tile is actually on the other side of the planet. I wonder how many new players assumed something is broken just like me, and just stopped. maybe hint at where the 2nd map is in the future

**Comment by Lesani** [comment]
And - I just noticed - looking at the screenshot above, the top-right corner does not show other sectors...
I have deleted all savegame data and re-played the first map after creating this issue, and on the 2nd playthrough I also found the 2nd map... looking at my screen now, I can see the first and 2nd map I played, and the 3rd upcoming as well
![image](https://user-images.githubusercontent.com/5182104/233805741-ecd7f6f4-cbd6-4f28-87cc-b3a0408acee4.png)

looking at the first screenshot it is clearly not there - so maybe there WAS an issue... just saying.

**Comment by Anuken** [comment]
> After all the coddling and guiding the game does the whole first level, one has to 1: assume the planet can be turned and 2: that the 2nd tile is actually on the other side of the planet. I wonder how many new players assumed something is broken just like me, and just stopped. maybe hint at where the 2nd map is in the future

The planet automatically pans to the next map once you unlock and view it, it's likely you interrupted the panning animation by accident. Considering you are the first one to have this problem, this is not likely to be a common issue.

> looking at the first screenshot it is clearly not there

The list only shows sectors you have landed on.

**Comment by ccope** [comment]
I also did not receive a notification that the level was complete, just a little logo in the top left saying "sector captured", which isn't clear to a new player that they should manually quit the level. Moving to the Planet Map view, which was unintuitive, showed a glitchy animation and left me looking at the Onset sector again. I did not see a way to rotate the map. Putting the second level on the direct opposite side of the world is also unintuitive.
